angelz510/react-challenge-project-sep-trial
# feat(order): added 'edit' and 'delete' functionality

## Changes
1. Users can now edit and delete an order from /view-orders

## Purpose
Before, users were unable to delete or edit an order. 

## Approach
Now, users can edit an orders, meal and quantity from viewOrders page.

## Pre-Testing TODOs
NA

## Testing Steps
First, add multiple orders.
Go to /view-orders and delete any order.
Change remaining orders name and quantity and click on update order.
Order should be updated.

## Learning
Research... reviews many 'to-do' apps. Conditionally rendering was key in breaking down the problem. I knew I needed an 'editing' stage and then render a form.

Feature Request: Add "edit" and "delete" order functionality
Closes #008
